homeassistant/components/bbb_gpio.py

```diff
@@ -63,7 +63,7 @@ def read_input(pin):
     """Read a value from a GPIO."""
     # pylint: disable=import-error,undefined-variable
     import Adafruit_BBIO.GPIO as GPIO
-    return GPIO.input(pin)
+    return True if GPIO.input(pin) is GPIO.HIGH else False
```

Comment:
Code readability is definitely important, so I'm glad you're thinking about that. 👍  It's just that once you get more familiarized with python you know that the `is` operator returns a boolean, so that construct takes longer to understand because you're looking for something that it's doing other than returning the boolean.